plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -406,12 +406,21 @@ String columnSchema()
     @Test
     void testRenameTable()
     {
-        assertUpdate("CREATE TABLE faker.default.original_table (id INTEGER, name VARCHAR)");
-        assertUpdate("ALTER TABLE faker.default.original_table RENAME TO faker.default.renamed_table");
+        assertUpdate("CREATE TABLE original_table (id INTEGER, name VARCHAR)");
+        assertUpdate("ALTER TABLE original_table RENAME TO renamed_table");
         // original_table should not exist anymore after renaming.
-        assertQueryFails("DESC faker.default.original_table", "line 1:1: Table 'faker.default.original_table' does not exist");
+        assertQueryFails("DESC original_table", "line 1:1: Table 'faker.default.original_table' does not exist");
         // should not allow renaming to an already existing table.
-        assertQueryFails("ALTER TABLE faker.default.renamed_table RENAME TO faker.default.renamed_table", "line 1:1: Target table 'faker.default.renamed_table' already exists");
-        assertUpdate("DROP TABLE faker.default.renamed_table");
+        assertQueryFails("ALTER TABLE renamed_table RENAME TO renamed_table", "line 1:1: Target table 'faker.default.renamed_table' already exists");
+        assertUpdate("DROP TABLE renamed_table");
+    }
+
+    @Test
+    void testRenameTableAcrossSchema(){
+        assertUpdate("CREATE SCHEMA new_schema");
+        assertUpdate("CREATE TABLE original_table (id INTEGER, name VARCHAR)");
```

Comment:
`original_table` is used in `testRenameTable`. Please don't use the same name. It will lead to flaky tests.